src/main/java/duke/task/TaskList.java

```diff
@@ -0,0 +1,213 @@
+package duke.task;
+
+import java.time.LocalDate;
+import java.time.format.DateTimeParseException;
+import java.util.ArrayList;
+
+import duke.exception.DukeException;
+
+/**
+ * A TaskList class to represent a list of tasks.
+ */
+public class TaskList {
+    private final ArrayList<Task> tasks;
+
+    /**
+     * Constructs a TaskList.
+     *
+     * @param tasks The list of tasks.
+     */
+    public TaskList(ArrayList<Task> tasks) {
+        this.tasks = tasks;
+    }
+
+    /**
+     * Checks if given String is not Numeric.
+     *
+     * @param str String to be checked.
+     * @return True if not numeric, False otherwise.
+     */
+    private static boolean isNotNumeric(String str) {
+        if (str == null || str.length() == 0) {
+            return true;
+        }
+        return !str.chars().allMatch(Character::isDigit);
+    }
```

Comment:
Names of methods should start with a verb. So maybe here it can be checkIfNumeric?